pages/demo-get-involved.vue

```diff
@@ -0,0 +1,237 @@
+                    class="ml-5 bg-white py-2 px-3 border border-gray-300 rounded-md shadow-sm text-sm leading-4 font-medium text-gray-700 hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
+                  >
+                    Change
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div
+            class="sm:grid sm:grid-cols-3 sm:gap-4 sm:items-start sm:border-t sm:border-gray-200 sm:pt-5"
+          >
+            <label
+              for="country"
+              class="block text-sm font-medium text-gray-700 sm:mt-px sm:pt-2"
+            >
+              Country
+            </label>
+            <div class="mt-1 mb-5 sm:mt-0 sm:col-span-2">
+              <select
+                id="country"
+                name="country"
+                autocomplete="country-name"
+                class="max-w-lg block focus:ring-indigo-500 focus:border-indigo-500 w-full shadow-sm sm:max-w-xs sm:text-sm border-gray-300 rounded-md"
+              >
+                <option>United States</option>
+                <option>Canada</option>
+              </select>
+            </div>
+          </div>
+          <div
+            class="sm:grid sm:grid-cols-3 sm:gap-4 sm:items-start sm:border-t sm:border-gray-200 sm:pt-5"
+          >
+            <label
+              for="about"
+              class="block text-sm font-medium text-gray-700 sm:mt-px sm:pt-2"
+            >
+              Why do you want to join?
+            </label>
+            <div class="mt-1 mb-5 sm:mt-0 sm:col-span-2">
+              <textarea
+                id="about"
+                name="about"
+                rows="3"
+                class="max-w-lg shadow-sm block w-full focus:ring-indigo-500 focus:border-indigo-500 sm:text-sm border border-gray-300 rounded-md"
+              />
+            </div>
+          </div>
+
+          <div
+            class="sm:grid sm:grid-cols-3 sm:gap-4 sm:items-start sm:border-t sm:border-gray-200 sm:pt-5"
+          >
+            <label
+              for="city"
+              class="block text-sm font-medium text-gray-700 sm:mt-px sm:pt-2"
+            >
+              City
+            </label>
+            <div class="mt-1 mb-5 sm:mt-0 sm:col-span-2">
+              <input
+                type="text"
+                name="city"
+                id="city"
+                autocomplete="address-level2"
+                class="max-w-lg block w-full shadow-sm focus:ring-indigo-500 focus:border-indigo-500 sm:max-w-xs sm:text-sm border-gray-300 rounded-md"
+              />
+            </div>
+          </div>
+
+          <div
+            class="sm:grid sm:grid-cols-3 sm:gap-4 sm:items-start sm:border-t sm:border-gray-200 sm:pt-5"
+          >
+            <label
+              for="region"
+              class="block text-sm font-medium text-gray-700 sm:mt-px sm:pt-2"
+            >
+              State / Province
```

Comment:
Please see the original form and use same fields.

Reply:
So no other fields like city zipcode country?

Reply:
same fields as in original form